utils/redis_reporter.go

```diff
@@ -0,0 +1,62 @@
+package utils
+
+import (
+	"fmt"
+	"strconv"
+
+	"gopkg.in/redis.v3"
+
+	"github.com/getlantern/golog"
+	"github.com/getlantern/measured"
+)
+
+var log = golog.LoggerFor("main")
+
+type redisReporter struct {
+	rc *redis.Client
+}
+
+func NewRedisReporter(redisAddr string) (measured.Reporter, error) {
+	rc := redis.NewClient(&redis.Options{
+		Addr: redisAddr,
+	})
+	_, err := rc.Ping().Result()
+	if err != nil {
+		return nil, fmt.Errorf("Unable to ping redis server: %s", err)
+	}
+	return &redisReporter{rc}, nil
+}
+
+func (rp *redisReporter) ReportError(s map[*measured.Error]int) error {
+	return nil
+}
+func (rp *redisReporter) ReportLatency(s []*measured.LatencyTracker) error {
+	return nil
+}
+func (rp *redisReporter) ReportTraffic(tt []*measured.TrafficTracker) error {
+	for _, t := range tt {
```

Comment:
measured will clear all data after each run of reporting. So current code is indeed not working as it uses HMSET